pilot/pkg/bootstrap/server.go

```diff
@@ -1281,3 +1288,73 @@ func (s *Server) initStatusManager(_ *PilotArgs) {
 		return nil
 	})
 }
+
+// initLocalSecretListener creates a new secret listener to listen for
+// secrets which represent new multi cluster participating clusters in a mesh.
+func (s *Server) initLocalSecretListener(args *PilotArgs) {
+	// Check if this is an instance of Istiod running as external istiod
+	// The logic is based on the following:
+	// 1. command istioctl secret create will always create a secret named
```

Comment:
`istioctl create-remote-secret` is just an example way to create the secret. Users can choose to create secrets for remote cluster without using istioctl.